tests/test_chargebee_all_fields.py

```diff
@@ -0,0 +1,318 @@
+            'total_in_local_currency',
+            'sub_total_in_local_currency',
+            'local_currency_code'
+        },
+        'payment_sources': {
+            'issuing_country',
+            'paypal',
+            'ip_address',
+            'bank_account',
+            'amazon_payment'
+        },
+        'transactions': {
+            'fraud_flag',
+            'authorization_reason',
+            'voided_at',
+            'reversal_txn_id',
+            'initiator_type',
+            'linked_payments',
+            'three_d_secure',
+            'merchant_reference_id',
+            'settled_at',
+            'reference_authorization_id',
+            'reversal_transaction_id',
+            'validated_at',
+            'fraud_reason',
+            'amount_capturable',
+            'reference_transaction_id'
+        },
+    }
+    # fields to remove for V2
+    fields_to_remove_V2 = {
+        'item_prices': {
+            'free_quantity_in_decimal',
+            'archivable',
+            'tax_detail',
+            'billing_cycles',
+            'trial_end_action',
+            'price_in_decimal',
+            'accounting_detail',
+            'shipping_period_unit',
+            'shipping_period',
+            'archived_at'
+        },
+        'invoices': {
+            'line_item_discounts',
+            'line_item_taxes',
+            'taxes',
+            'discounts',
+            'dunning_status',
+            'vat_number'
+        },
+        'credit_notes': {
+            'voided_at',
+            'vat_number',
+            'discounts'
+        },
+        'items': {
+            'archivable',
+            'gift_claim_redirect_url',
+            'applicable_items',
+            'redirect_url',
+            'usage_calculation',
+            'included_in_mrr'
+        },
+        'coupons': {
+            'invoice_notes',
+            'meta_data',
+            'archived_at'
+        },
+        'customers': {
+            'backup_payment_source_id',
+            'meta_data',
+            'cf_company_id',
+            'custom_fields',
+            'created_from_ip',
+            'consolidated_invoicing',
+            'billing_day_of_week',
+            'vat_number'
+        },
+        'subscriptions': {
+            'cancel_reason',
+            'start_date',
+            'meta_data',
+            'remaining_billing_cycles',
+            'payment_source_id',
+            'custom_fields',
+            'item_tiers',
+            'invoice_notes',
+            'created_from_ip',
+            'cancel_reason_code',
+            'coupon',
+            'coupons'
+        },
+        'transactions': {
+            'error_text',
+            'reference_number',
+            'error_code',
+            'refunded_txn_id'
+        },
+        'promotional_credits': {
+            'reference'
+        },
+        'events': {
+            'user'
+        }
+    }
+    # fields to remove for V1
+    fields_to_remove_V1 = {
```

Comment:
Provide a reason for skipping in the comment.

Reply:
I have added general comments for all the streams [here](https://github.com/singer-io/tap-chargebee/pull/83/files#diff-1f08d51aa03a23c2c59e9bec09f6da13b0bbede459fe4a8a3a5979e05ea79892R6).

Reply:
Updated the comment for removal of fields in all fields test.